### ISSUE TITLE
[Java] [WARNING] The rule GuardLogStatement is referenced multiple times in "pmd". Only the last rule configuration is used.

### ISSUE THREAD

**Issue by patpatpat123** [pmd/pmd#3326]
Hello PMD Team,

Apologies for the trouble, but this warning is 100% reproducible.

I am currently using PMD Maven plugin 3.14.0, with SonarQube 8.9 with PMD version 3.3.1 (hence PMD 6.30).

```
  <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-pmd-plugin</artifactId>
                <version>3.14.0</version>
                <configuration>
                    <targetDirectory>.out/reports/pmd</targetDirectory>
                    <outputDirectory>target/reports/pmd</outputDirectory>
                </configuration>
            </plugin>
```

One each analysis, I am getting 100%
 
```[WARNING] The rule GuardLogStatement is referenced multiple times in "pmd". Only the last rule configuration is used.```

Could you please help check why this particular rule is referenced multiple times in PMD please?

Thank you

**Comment by adangel** [comment]
Hi @patpatpat123 ,
thanks for the report.
Would you mind to share the other configurations as well? (sonarqube, pmd ruleset in use).

The piece you shared is the maven-pmd-plugin configuration - there you don't specify a [rulesets](https://maven.apache.org/plugins/maven-pmd-plugin/pmd-mojo.html#rulesets) element, which means, you use the plugin's default ruleset. This [default ruleset](https://maven.apache.org/plugins/maven-pmd-plugin/examples/usingRuleSets.html#The_default_ruleset) does not have the rule GuardLogStatement enabled, so you must use somehow a different ruleset.

**Comment by jsotuyod** [comment]
Marking this as stale as it's been over 3 years without feedback from the reporter nor a clear way to reproduce it.
